functions/AutoRemediateS3-002.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
+
+const config = require('./config');
+const AWS = require("aws-sdk");
+
+const CCRuleCode = 'S3-002'
+const CCRuleName = 'BucketPublicReadAcpAccess'
+const allUsersURI = 'http://acs.amazonaws.com/groups/global/AllUsers'
+const readAcpPermission = "READ_ACP"
+
+module.exports = {
+
+  removeAcpPermission: function (thisGrant, newAcl) {
```

Comment:
Don't mutate input parameters.

Prefer to return a copy.

Ref: Clean Code - avoid output arguments.